_data/vendors.yml

```diff
@@ -41,6 +41,14 @@
   pricing_source: https://checklyhq.com/pricing/
   updated_at: 2019-07-30
 
+- name: CloudFlare
+  url: https://www.cloudflare.com
+  base_pricing: $20
```

Comment:
Would suggest that given they have a free offering that the free offering is their "Base Price"